### PR TITLE
feat(ui): Add dropdown and autocomplete for more search parameters

### PR DESCRIPTION
**Pull request by MeredithAnya**
ezgif com-video-to-gif 1

cc @ehfeng @getsentry/design

### CODE REVIEW

File: src/sentry/static/sentry/app/stores/streamTagStore.jsx [hunk 1]
```diff
@@ -55,6 +55,30 @@ const StreamTagStore = Reflux.createStore({
...
+      lastSeen: {
+        key: 'lastSeen',
+        name: 'Last Seen',
+        values: ['-1h', '+1d', '-1w'],
+        predefined: true,
+      },
+      firstSeen: {
+        key: 'firstSeen',
+        name: 'First Seen',
+        values: ['-1h', '+1d', '-1w'],
```
Comment (ckj) [inline src/sentry/static/sentry/app/stores/streamTagStore.jsx]: How do `+` values work for these? Wouldn't that be saying "show me things that were first or last seen tomorrow" or am I misunderstanding?

Comment (MeredithAnya) [reply src/sentry/static/sentry/app/stores/streamTagStore.jsx]: I'm pretty sure +1h would be anything seen one hour or more ago vs. -1h would be anything seen in the last hour

Comment (ckj) [reply src/sentry/static/sentry/app/stores/streamTagStore.jsx]: Ah, that makes much more sense, though I do wonder why it doesn't use the same `<, >, =` as the timestamp syntax does. Nice to have this stuff surfaced, regardless.

**Comment by benvinegar** [comment]
Nice. 🙌

File: src/sentry/static/sentry/app/stores/streamTagStore.jsx (hunk 1 above)
Comment (MeredithAnya) [reply src/sentry/static/sentry/app/stores/streamTagStore.jsx]: was maybe going to link https://docs.sentry.io/learn/search/ in der somewhere but didn't know best way to do that

Comment (billyvg) [reply src/sentry/static/sentry/app/stores/streamTagStore.jsx]: It would be nice to have more descriptive labels for these as it doesn't seem intuitive.

**Review by billyvg — APPROVED** [review]
👍

**Pull request merged**